CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Added
+* Added validation of the master availability to the `utils.get_space`
+  method before receiving the space through the connection (#95 #331).
```

Comment:
After discussion, I think we should implement master waiting with timeout or not close #95 here (we may refer to it with `Part of #95` since it partially solves it).